indico/modules/events/editing/client/js/editing/timeline/selectors.js

```diff
@@ -173,14 +173,15 @@ export const getStaticData = state => state.staticData;
 
 export const getFileTypes = createSelector(
   getStaticData,
-  staticData => {
+  getDetails,
```

Comment:
Isn't this a dangerous change?

There's an exception when I click `EditableSubmissionButton` because it also uses `getFileTypes` to pass it as prop to the `FileManager` but I don't think the store contains any timeline details, since we're passing the contribution code as `staticData`. Perhaps we could find a balance here?

Reply:
That's a nice observation I didn't notice, thanks. I'll find a way to fix this.